internal/integration/infra_sdk_emitter.go

```diff
@@ -0,0 +1,181 @@
+package integration
+
+import (
+	"fmt"
+	"math"
+	"time"
+
+	"github.com/newrelic/infra-integrations-sdk/data/metric"
+	"github.com/newrelic/infra-integrations-sdk/integration"
+	"github.com/newrelic/newrelic-telemetry-sdk-go/cumulative"
+	"github.com/newrelic/nri-prometheus/internal/pkg/labels"
+	dto "github.com/prometheus/client_model/go"
+	"github.com/sirupsen/logrus"
+)
+
+// InfraSdkEmitter is the emitter using the infra sdk to output metrics to stdout
+type InfraSdkEmitter struct {
+	deltaCalculator *cumulative.DeltaCalculator
+}
+
+// NewInfraSdkEmitter creates a new Infra SDK emitter
+func NewInfraSdkEmitter() (*InfraSdkEmitter, error) {
+	return &InfraSdkEmitter{
+		// perhaps allow configuration..
+		deltaCalculator: cumulative.NewDeltaCalculator(),
```

Comment:
This works when the integration is long-running, does it work as well if it terminates?
Does it stores the values on file?

Reply:
No it does not store anything in files, it's memory only. 
It's probably not going to work very well because we're going to run this only once so the values in memory will be lost.
I don't have a good approach to fix this. Only thing I can think of is using the SDK "storer" to keep the value in a file and replicate the implementation of the deltaCalculator. 
What do you think?

Reply:
We could also read the stored values and "simulate" we receive those metrics in order to initialise the deltaCalculator. Otherwise I have the feeling that that the deltas will not work as expected

Reply:
couldn't we cherry pick the storer implementatino from the master sdk branch and use that? I guess that the sdk storer is going to be needed for the integrations already using it.

Reply:
I believe we removed it because the agent would be responsible to keep that data to calculate deltas. I'll recover the storer implementation, that's probably easier to do.

Reply:
I've added an implementation here. But it's possible that this will be done in the agent.

Reply:
I do not see any new commit, are you going to include the implementation in this PR?

Reply:
@paologallinaharbur sorry, i had it in another branch and didn't merge it to my repo main. You should see it now

Reply:
Removed that commit because we've agreed that the delta calculation will be done in the agent. We'll still need a few changes because the way we send the histogram and summary metrics is most likely going to change